tests/CocktailParty.tex

```diff
@@ -9,46 +9,45 @@ \subsection{Focus}
+	The calling person introduces themself by name before giving the order of a drink.
+	The robot leads the dialogue to learn the person and retrieve their drink order. \\
 
-	\item \textbf{Sitting person:} At least one person is sitting and minding their own business (i.e. not looking directly to the robot). The robot must locate that person and ask if the person wants something to drink. The robot must also ask for the person's name and memorize them (i.e. execute a learning procedure of the name and the person's features). \\
 
-	\textbf{Remark:} At least one sitting person has an order to place. No sitting person will attend to the robot's call
-	(i.e. robot has to approach to them).
+	\item \textbf{Taking the order:} After the robot has took the order of the first guest, it can either take more orders or proceed to place the order.
 
-	\item \textbf{Placing the orders:} The robot has to navigate to the \textit{Bar}, a designated location in another room where drinks are served. The robot must repeat each order to the \textit{Barman}, clearly stating:
+	\item \textbf{Placing the orders:} The robot has to navigate to the \textit{Bar}. The robot must repeat each order to the \textit{Barman}, clearly stating:
 	\begin{enumerate}
 		\item The person's name,
 		\item The person's chosen drink,
 		\item A description of unique characteristics of that person that allow the \textit{Barman} to find them (e.g. gender, hair colour, how is dressed, etc).
 	\end{enumerate}
 
-	While the robot places the orders, the people in the \quotes{party room} may change their places within the party room (on request of the referees).
+	While the robot places the orders, the people in the \quotes{party room} change their places within the party room (on request of the referees).
 
-	\item \textbf{Missing beverage:} One of the ordered drinks is not available, therefore, missing from the bar. The robot should realize this inconvenience and tell the \textit{Barman}, providing a list of 3 alternatives considering the other drinks it needs to deliver. If the robot can't detect which drink is missing, the \textit{Barman} will clearly state which of the beverages is not available and provide a list of 3 alternatives.
+	\item \textbf{Missing beverage:} One of the ordered drinks is not available, therefore, missing from the bar.
+	The robot should realize this inconvenience and tell the \textit{Barman}, providing a list of 3 alternatives considering the other drinks it needs to deliver.
+	If the robot can't detect which drink is missing, the \textit{Barman} will clearly state which of the beverages is not available and provide a list of 3 alternatives.
```

Comment:
If the robot cannot detect which drink is missing, the \textit{Barman} will clearly state which of the beverages is not available and provide a list of 3 alternatives.